src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/CalculationController.kt

```diff
@@ -195,6 +198,30 @@ class CalculationController(
     return calculationTransactionalService.findCalculationResults(calculationRequestId)
   }
 
+  @GetMapping(value = ["/detailed-results/{calculationRequestId}"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'RELEASE_DATES_CALCULATOR')")
+  @ResponseBody
+  @Operation(
+    summary = "Get release dates for a calculationRequestId with additional details",
+    description = "This endpoint will return the release dates based on a calculationRequestId along with hints and full descriptions.",
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "200", description = "Returns calculated dates"),
+      ApiResponse(responseCode = "401", description = "Unauthorised, requires a valid Oauth2 token"),
+      ApiResponse(responseCode = "403", description = "Forbidden, requires an appropriate role"),
+      ApiResponse(responseCode = "404", description = "No calculation exists for this calculationRequestId"),
+    ],
+  )
+  fun getDetailedResults(
+    @Parameter(required = true, example = "123456", description = "The calculationRequestId of the results")
+    @PathVariable("calculationRequestId")
+    calculationRequestId: Long,
+  ): DetailedCalculationResults {
+    log.info("Request received return calculation results for calculationRequestId {}", calculationRequestId)
```

Comment:
Might be nice to differentiate this from the out log outputs of which there are 4 the same atm.